lib/ansible/vars/manager.py

```diff
@@ -446,7 +445,7 @@ def plugins_by_groups():
         else:
             return all_vars
 
-    def _get_magic_variables(self, play, host, task, include_hostvars, include_delegate_to, _hosts=None, _hosts_all=None):
+    def _get_magic_variables(self, play, host, task, include_hostvars, _hosts=None, _hosts_all=None):
```

Comment:
Note: `include_delegate_to` is not used by `_get_magic_variables`. This is an unrelated change, but just for clarity I removed it.